Source/g_game.c

```diff
@@ -714,12 +709,8 @@ static void G_ReloadLevel(void)
     G_BeginRecording();
   }
 
-  // force players to be initialized upon first level load
-  for (i = 0; i<MAXPLAYERS; i++)
-   if (playeringame[i])
-    players[i].playerstate = PST_REBORN;
-
-  G_DoLoadLevel();
+  G_InitNew(gameskill, gameepisode, gamemap);
```

Comment:
Isn't this the route that resets `netgame` to `false`?

Reply:
`netgame` is resets in `G_DoNewGame`. I tested this branch it seems to work.